### ISSUE TITLE
Once there is a cime tag with pio2 as the default, update to it

### ISSUE THREAD

**Issue by billsacks** [ESCOMP/CTSM#1175]
We should run the full aux_clm test suite after updating to the cime tag with pio2 as the default, to verify that all tests pass and are bit-for-bit.

The relevant cime PR is https://github.com/ESMCI/cime/pull/3732. This is currently planned for cesm2_3_beta01.

**Comment by ekluzek** [comment]
And run the ctsm_sci test list as well. And I think you should run the fates test list.

You could also run mosart and rtm, but if the other tests work I can't imagine having one of them showing something different.

**Comment by billsacks** [comment]
We should also check the performance of many tests against the baseline to make sure that there aren't cases where we're seeing significantly degraded performance. We can do this by paying attention to TPUTCOMP (which I typically ignore), maybe together with spot-checking the model initialization time (which will include init_interp time) for a few tests (because I don't think that model initialization time is considered in TPUTCOMP). Alternatively, we could compare the RUN time from the TestStatus files from a test suite run with pio1 vs. one run with pio2.

**Comment by billsacks** [comment]
Blocks #1194

**Comment by billsacks** [comment]
Getting to this now....

Regarding some of the above comments:

- I did a bunch of careful performance checks back in October, when I was doing extensive pio2 testing. So I'm not going to redo those extensive performance checks with the cime update.

- I also ran the `ctsm_sci` and `fates` test lists with pio2 back in October. There were some issues with both, but seemingly unrelated to pio2. Again, I'm not going to redo that testing in this cime update, because I believe that many / all of the issues are still outstanding.
